keycloak/sms-provider/src/main/java/org/sunbird/keycloak/KeycloakSmsAuthenticatorConstants.java

```diff
@@ -6,6 +6,8 @@
 public class KeycloakSmsAuthenticatorConstants {
     public static final String ATTR_MOBILE = "mobile";
     public static final String ANSW_SMS_CODE = "smsCode";
+    public static final String COUNTRY_CODE = "+91";
```

Comment:
This must be configuration not code. Please add a TODO.

Reply:
@kochhar Ok, I'll add that to the attributes of the configurations. But what if user does not set these? Should we have them as default ones?